ListModeration.md

```diff
@@ -0,0 +1,22 @@
+# List Moderation Guidance
+
+Every email list creates an obligation to ensure that it is not used for harmful purposes.
+By default, a list's moderators are the group's chairs, the list's maintainers or W3C administrators, 
+but specific individuals may also be appointed as moderators.  The purpose of list moderation is to ensure that the list is used in keeping with W3C's mission and values.
+
+## Moderator Guidelines
+
+W3C list moderation typically requires only a very light touch.
+
+Main responsibilities:
+- Follow all posted content, at least enough to notice whether any content violates W3C policies.
+- Ensure that list participants abide by the [W3C Code of Professional Conduct](https://www.w3.org/Consortium/cepc/) (CPC).  The enforcement approach should depend on the circumstances:
```

Comment:
CEPC, unless/until we decide to change name.

Reply:
Done in the latest version.